src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -51,22 +54,25 @@ export default class ItemDescription extends React.PureComponent<Props, State> {
   render() {
     return this.state.health ? (
       <Row>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           <strong>Health: </strong>
           <HealthIndicator id={this.props.item.name} health={this.state.health} mode={DisplayMode.SMALL} />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           {!this.props.item.istioSidecar && <MissingSidecar />}
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4}>
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
           <strong>Config: </strong>{' '}
           <ConfigIndicator
             id={this.props.item.name + '-config-validation'}
             validations={[this.props.item.validation]}
             size="medium"
           />
         </Col>
-        <Col xs={12} sm={12} md={4} lg={4} />
+        <Col xs={12} sm={12} md={this.state.columnWidth} lg={this.state.columnWidth}>
+          {this.props.item.apiType && <strong>Api: </strong>}
+          {this.props.item.apiType && <ApiTypeIndicator apiType={this.props.item.apiType} />}
```

Comment:
And here both could be only one conditional (hopefully on the top element level).

Reply:
I tried with one conditional, but i had an error saying I needed one parent. Can you suggest something in the code ? I know it's not great as it is

Reply:
You can wrap the elements like this: `<>Element here</>`.